CMakeLists.txt

```diff
@@ -67,10 +66,15 @@ set(ENFTUN_SRCS
     src/tcp.c
     src/tcp_multi.c
     src/tls.c
+    src/tls_tpm.c
     src/tun.c
     src/udp.c
     )
 
+if(USE_TSS_ENGINE)
```

Comment:
Need to remove these lines, no longer relevant.

Reply:
Whoops, removed.